redis/commands/timeseries/info.py

```diff
@@ -67,8 +67,8 @@ def __init__(self, args):
         self.total_samples = response["totalSamples"]
         self.labels = list_to_dict(response["labels"])
         self.retention_msecs = response["retentionTime"]
-        self.lastTimeStamp = response["lastTimestamp"]
-        self.first_time_stamp = response["firstTimestamp"]
+        self.last_timestamp = response["lastTimestamp"]
```

Comment:
Just in case the server pukes, maybe we what to think about doing response.get("lastTimestamp") (etc) for everything. This would ensure that we don't have to necessarily break in the future.